hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/PartitionBucketIndexManager.scala

```diff
@@ -195,7 +196,8 @@ class PartitionBucketIndexManager extends BaseProcedure
       logInfo("Perform OVERWRITE with dry-run disabled.")
       val partitionsToRescale = rescalePartitionsMap.keys
       // get all fileSlices need to read
-      val allFilesMap = FSUtils.getFilesInPartitions(context, metaClient, HoodieMetadataConfig.newBuilder.enable(mdtEnable).build,
+      val metadataConfig = HoodieMetadataConfig.newBuilder.enable(mdtEnable).build
+      val allFilesMap = FSUtils.getFilesInPartitions(context, metaClient, metadataConfig,
```

Comment:
Redundant change?

Reply:
Yes I think this an older change let me revert this line back